app/views/admin/loans/content/_table_question.html.slim

```diff
@@ -0,0 +1,89 @@
+.table-responsive.editable-tables
+
+  / TODO: Convert input to expected input for loan question
+  input type="hidden" data-container="breakeven"
+
+  .table-container
+    h4.title = t("loan.breakeven.products", count: 2)
+
+    table.table.table-condensed.table-striped.table-bordered.editable-table data-table="products"
+      thead
+        tr
+          th = t("loan.breakeven.product_name")
+          th.wide = t("loan.breakeven.description")
+          th = t("loan.breakeven.unit")
+          th.narrow = t("loan.breakeven.price")
+          th.narrow = t("loan.breakeven.cost")
+          th.narrow = t("loan.breakeven.quantity")
+          th.narrow: .actions: a data-action="add": i.fa.fa-plus-circle
+      tbody
+        / Hidden row is used to generate new rows
+        / Is at top instead of bottom to not affect striped rows
+        tr.ui-state-default.hidden data-group="product"
+          td: textarea data-input="name" placeholder="Name of product"
+          td: textarea data-input="description" placeholder="Description"
+          td: textarea data-input="unit" placeholder="E.x.: pair of pants"
+          td: input data-input="price" type="number" placeholder="100"
+          td: input data-input="cost" type="number" placeholder="10"
+          td: input data-input="quantity" type="number"
+          td: .actions
+            a.delete data-action="delete" title="Remove row": i.fa.fa-trash
+            a.move data-action="move" title="Click to interact with row.": i.fa.fa-arrows-v
+
+        - breakeven_data[:products].each do |product|
+          tr.ui-state-default data-group="product"
+            td: textarea data-input="name" = product[:name]
+            td: textarea data-input="description" = product[:description]
+            td: textarea data-input="unit" = product[:unit]
+            td: input data-input="price" value== product[:price] type="number"
+            td: input data-input="cost" value== product[:cost] type="number"
+            td: input data-input="quantity" value== product[:quantity] type="number"
+            td: .actions
+              a.delete data-action="delete": i.fa.fa-trash
+              a.move data-action="move": i.fa.fa-arrows-v
+
+        tr.ui-state-default data-group="product"
+          td: textarea data-input="name" placeholder="Name of product"
```

Comment:
Placeholders and other are not yet translated. I am not sure if placeholders are desired and will check in with Roxana regarding visual review of feature.